### ISSUE TITLE
[Sprint] Use f-strings instead .format() function on asv/commands/rm.py

### ISSUE THREAD

**Issue by LucyJimenez** [airspeed-velocity/asv#1181]
xref [#909](https://github.com/airspeed-velocity/asv/issues/909)

Since we support Python > 3.7 now, we can make use of f-strings, to make code more readable.

As an example, the next asv code:
`machine_env_name = "{}/{}".format(machine, env_name)`
or the next, which is equivalent:
`machine_env_name = "{0}/{1}".format(machine, env_name)`
Could be written as:
`machine_env_name = f"{machine}/{env_name}"`

(note the `f` before the string quotes)

---

The next command returns the list of strings to replace: 

 `grep -R --exclude-dir="build" --exclude="*.css" ".format(" asv/commands/rm.py`

For this file, you will find the following issues to solve:
```
asv/commands/rm.py:                raise util.UserError("Invalid pattern '{0}'".format(pattern))
asv/commands/rm.py:                        "'{0}' appears more than once".format(parts[0]))
asv/commands/rm.py:            log.info("Removing {0} benchmarks in {1} files".format(
asv/commands/rm.py:            log.info("Removing {0} files".format(len(files_to_remove)))
```

**Comment by Daniet** [comment]
@LucyJimenez 
i will work on this

**Comment by LucyJimenez** [comment]
Closed #1186
